types/ckeditor__ckeditor5-utils/src/translation-service.d.ts

```diff
@@ -1,4 +1,4 @@
-export interface GetPluralForm {
+interface GetPluralForm {
```

Comment:
This is another case where its unclear as to why you wouldn't export this interface.